### ISSUE TITLE
direction effect and results pleio

### ISSUE THREAD

**Issue by sevi2018** [cuelee/pleio#7]
Hi,
I'm running pleio including more than 15 GWASs,
I have noticed that if I flip the beta  of some of the GWASs i get different results,
I still did confirm the loci detected in the first analysis realized using the original beta sign, but on top of that I get more loci that reach genome-wide significance.
My question is, is this what you expect to see? 
And in case it is can you explain me why?

Thanks in advance for your help,

**Comment by cuelee** [comment]
Yes, reversing the beta of some (but not all) of the inputs will produce a different result, but in a PLEIO analysis, changing the direction of the input beta is against the model assumption.
